src/osx/carbon/cursor.cpp

```diff
@@ -277,7 +279,26 @@ wxCursor::wxCursor(const wxString& cursor_file, wxBitmapType flags, int hotSpotX
     if ( flags == wxBITMAP_TYPE_MACCURSOR_RESOURCE )
     {
 #if wxOSX_USE_COCOA
-        wxFAIL_MSG( wxT("Not implemented") );
+        wxImage image;
```

Comment:
Just curious: So we no longer care about `wxUSE_xx` defines in wxWidgets code (see `wxUSE_IMAGE` below)?

Reply:
I don't know. But I will add them here.